src/components/FlagButton/FlagButton.test.tsx

```diff
@@ -5,17 +5,26 @@ import '@testing-library/jest-dom'
 
 describe('components/FlagButton', () => {
   test('should have aria-expanded to true', () => {
-    render(<FlagButton isFlagsMenuOpened isoCode="FR" />)
+    render(<FlagButton isFlagsMenuOpened isoCode="FR" countryName="France" />)
     expect(screen.getByRole('button')).toHaveAttribute('aria-expanded', 'true')
   })
 
   test('should have aria-expanded to false', () => {
-    render(<FlagButton isFlagsMenuOpened={false} isoCode="FR" />)
+    render(
+      <FlagButton isFlagsMenuOpened={false} isoCode="FR" countryName="France" />
```

Comment:
I think the best approach would be to get the country name inside the Flag component, but for that we would need to pass "langOfCountryName" for all components until reach that component, which is something I tried to avoid, and that's the reason why I'm passing only the countryName and isoCode for the FlagButton component.